make_ghpages/requirements.txt

```diff
@@ -1,2 +1,2 @@
 jinja2==3.0.0a1
-optimade>=0.12.0
+optimade==0.12.5
```

Comment:
```suggestion
optimade==0.12.6
```

Reply:
I was hoping dependabot would do this bit, but take your point on the actual pinning

Reply:
Right. You want to test the setup. No problem 👍